### PR TITLE
libirecovery 1.1.0

### PR DESCRIPTION
**Pull request by p-linnane**
Created by `brew bump`

---

Created with `brew bump-formula-pr`.

### CODE REVIEW

**Comment by p-linnane** [comment]
This requires `libimobiledevice-glue` which we do not package. While writing a formula for it, I found it requires v2.3.0 of `libplist` (https://github.com/Homebrew/homebrew-core/pull/128974), which is blocked by a new release of `libimobiledevice`.

**Comment by chenrui333** [comment]
> This requires `libimobiledevice-glue` which we do not package. While writing a formula for it, I found it requires v2.3.0 of `libplist` (#128974), which is blocked by a new release of `libimobiledevice`.

we actually had a PR for libimobiledevice-glue, #87059

**Comment by p-linnane** [comment]
> we actually had a PR for libimobiledevice-glue, #87059

It has a tagged release now, and I've written it in our modern format. Hopefully we get a new `libimobiledevice` soon (upstream said it was coming) and then can unravel this cascading mess.

File: Formula/libirecovery.rb [hunk 1]
```diff
@@ -25,17 +25,17 @@ class Libirecovery < Formula
...
   on_linux do
-    depends_on "pkg-config" => :build
     depends_on "libusb"
     depends_on "readline"
   end
 
   def install
     system "./autogen.sh" if build.head?
-    system "./configure", "--disable-dependency-tracking",
+    system "./configure", *std_configure_args.reject { |arg| arg == "--disable-debug" },
```
Comment (p-linnane) [inline Formula/libirecovery.rb:37]: @SMillerDev I didn't know this was a thing. Gotta start using that going forward for the ones that complain. Thanks!

**Comment by p-linnane** [comment]
Closing as there has been no response from upstream.

**Comment by chenrui333** [comment]
#144336

**Pull request closed without merging**